scripts/x-charts-pro.exports.json

```diff
@@ -55,8 +55,8 @@
   { "name": "CartesianSeriesType", "kind": "TypeAlias" },
   { "name": "ChartContainerPro", "kind": "Variable" },
   { "name": "ChartContainerProProps", "kind": "Interface" },
-  { "name": "ChartDataProvider", "kind": "Function" },
-  { "name": "ChartDataProviderProps", "kind": "TypeAlias" },
+  { "name": "ChartDataProviderPro", "kind": "Function" },
+  { "name": "ChartDataProviderProProps", "kind": "TypeAlias" },
   { "name": "ChartDrawingArea", "kind": "TypeAlias" },
```

Comment:
I would be in favor of adding an export of the MIT container in `packages/x-charts-pro/src/index.ts` such that pro users can both have the provider with and without zoom